redis-cluster/example/config/redis-env

```diff
@@ -0,0 +1,5 @@
+REDIS_NODE_NAME=foo-redis
+REDIS_HEADLESS_SERVICE=foo-redis
+NODE_REPLICA_COUNT=9
+REDIS_REPLICA_COUNT=2
```

Comment:
How is REDIS_REPLICA_COUNT used?

Reply:
Its how it knows how many replicas to setup and how to bootstrap the cluster. 
It has to build a cluster create with the relevant nodes.

Reply:
I'm asking, because it doesn't seem to be standard Redis ENV, and it doesn't seem to be used in the scripts

Reply:
Also, I got the same amount of masters (4 in 9 node cluster) while running `REDIS_REPLICA_COUNT=2` and `REDIS_REPLICA_COUNT=3` or `REDIS_REPLICA_COUNT=1`

Reply:
Good spot, I fixed this in the base before and forgot to push it. Ive now pushed a fix.